networkml/parsers/pcap/pcap_utils.py

```diff
@@ -9,9 +9,9 @@
 import netaddr
 from scapy.layers.l2 import Ether
 from scapy.layers.inet import IP, ICMP
-from scapy.layers.inet6 import *
+import scapy.layers.inet6
```

Comment:
why not `from scapy.layers.inet6 import IPv6` ?

Reply:
Because we need to import all the scapy IPv6 header parsers (else acapy will fail to parse most any IPv6 packet). The IPv6 layer is not enough by itself. It's not consistent with IPv4 where don't need the TCP header etc.